README.md

```diff
@@ -178,6 +178,23 @@ end
 
 See [examples/market](examples/market/marketbot.rb) for a working example.
 
+### Matching text in message attachments
+
+You can respond to text in [attachments](https://api.slack.com/docs/message-attachments) with 
+`attachment`. It will scan `text`, `pretext` and `title` fields in each attachment until a first
+match is found.
+
+For example you can match [this example attachment](http://goo.gl/K0cLkH)
+by its `title` with the following bot:
+
+```ruby
+class Attachment < SlackRubyBot::Bot
+  attachment 'Slack API Documentation' do |client, data, stocks|
```

Comment:
What's `stocks` here? I think what we want is to pass in the matching attachment into this block.

Reply:
Copy-paste issue 😃will fix this, thanks